CMakeLists.txt

```diff
@@ -408,7 +439,7 @@ if(CLR_CMAKE_PLATFORM_UNIX)
 add_definitions(-DDISABLE_CONTRACTS)
 # The -ferror-limit is helpful during the porting, it makes sure the compiler doesn't stop
 # after hitting just about 20 errors.
-add_compile_options(-ferror-limit=4096)
+add_compile_options(-ferror-limit=40)
```

Comment:
Sure I will remove this. It was useful for me when I only cared about the first few errors, as the rest were just derivatives.